shell/plugins/i18n.js

```diff
@@ -58,8 +58,6 @@ export function directiveSsr(vnode, binding) {
 const i18n = {
   name:    'i18n',
   install: (Vue, _options) => {
-    _options?.store?.dispatch('i18n/init');
```

Comment:
Vue plugins can't have a blocking async install method otherwise I would've just made the modification here.